src/core/mkContainerFromCabalExe.nix

```diff
@@ -0,0 +1,74 @@
+{ repoRoot, iogx-inputs, user-inputs, pkgs, lib, system, ... }:
+
+sourceRepoUrl:
+userConfig':
+
+let
+  utils = lib.iogx.utils;
+
+  inherit (iogx-inputs.nix2container.packages.${system}) nix2container;
```

Comment:
the `${system}` bit shouldn't be necessary. can you check please?

Reply:
I guess it wasn't necessary. No clue why. Remove.